src/models/servercountrymodel.cpp

```diff
@@ -127,6 +127,13 @@ bool ServerCountryModel::fromJsonInternal(const QByteArray& s) {
   return true;
 }
 
+void ServerCountryModel::writeSettings() {
```

Comment:
I'm noticing that all of the `FooModel::writeSettings()` are kind of exactly the same thing for every model. Is this something that we could just implement in the `BaseModel` class?